partials/Dockerfile.r-ver.partial

```diff
@@ -0,0 +1,27 @@
+# Add the Rocker install scripts
+COPY scripts /rocker_scripts
```

Comment:
I still do think we should copy scripts over individually as needed, which would play nicely with caching.  

Copying them all here is kinda pointless since we still have to repeat this in the downstream Dockerfiles if we are making separate Dockerfiles anyway, right?

Reply:
We touched on this before.  Ot does play better with caching when developing, not as much once things are stable.  I think it's much better for users who are customizing to have all of these scripts available to them as long as they are building on top any `r-ver` or derived image.